httpreq/httpreq.go

```diff
@@ -167,3 +167,38 @@ func (r *HttpReq) call(v interface{}, httpClient *http.Client) (int, error) {
 	return resp.StatusCode, nil
 
 }
+
+func (r *HttpReq) RawCall() (*http.Response, error) {
+	return r.rawCall(defaultClient)
+}
+
+func (r *HttpReq) RawCallWithClient(httpClient *http.Client) (*http.Response, error) {
+	return r.rawCall(httpClient)
+}
+
+func (r *HttpReq) RawCallWithTransport(transport *http.Transport) (*http.Response, error) {
+	httpClient := &http.Client{Transport: transport}
+	return r.rawCall(httpClient)
+}
+
+func (r *HttpReq) SetRawGlobalTransport(transport *http.Transport) (*http.Response, error) {
+	if defaultClient != nil {
+		defaultClient.Transport = transport
+	}
+	return r.rawCall(defaultClient)
```

Comment:
This method is not necessary. 
I think `RawCallWithTransport` method is enough.